bioconda_utils/build.py

```diff
@@ -240,11 +241,15 @@ def build_recipes(
 
     logger.info('blacklist: %s', ', '.join(sorted(blacklist)))
 
+    get_recipes = utils.get_recipes
```

Comment:
I mean this could be in an else, belonging to the if below.
```
if latest_only:
    get_recipes = utils.get_latest_recipes
else:
    get_recipes = utils.get_recipes
```